bots/bots-architecture/README.md

```diff
@@ -37,31 +50,52 @@ emitted by the server's streamer. Bots subscribe to the collection of messages
 for their own user account, creating a stream that is updated every time a
 message is sent either directly to them or any room they are joined in.
 
-## Rocket.Chat JS SDK
+## Rocket.Chat SDK
+
+The SDK is a low-level software module that offers an interface for external
+consumers to subscribe to message streams, send messages, and call methods on the
+Rocket.Chat server via WebSockets / DDP.
+
+Bot framework adapter are written using the SDK.
+
+## Framework Adapter
 
-The SDK is a Node.js module that provides an interface for external applications
-to subscribe to message streams, send messages and call methods on the
-Rocket.Chat server (via WebSockets / DDP). It would be a dependency of most bots
-built on Node.js and we're working on a Python translation as well.
+An adapter (also referred as **connector** or **middleware**) is a software component
+that is used to link the framework with the Rocket.Chat host.
 
-## Framework Adapters
+An adapter allows bot creators to design and build bot interactions according to the
+framework's rules and practices without dealing with the low-level programming
+details of the communication with the Rocket.Chat host.
 
-Adapters, sometimes referred to as **connectors** or **middleware**, provide the
-parsing of message schemas and methods between the chatbot framework and the
-Rocket.Chat SDK. Once loaded into a project, they allow bot creators to compose
-interactions in the framework's terms, ignoring the specific requirements of
-Rocket.Chat.
+Adapters use Rocket.Chat SDK to communicate with the Rocket.Chat host.
 
-## Bot Frameworks
+## Bot Framework
 
-Bot frameworks such as Hubot, Botkit, Rasa and Botpress each provide unique
-approaches to composing conversational interfaces. Some run their own hosting
-for bots, some just provide the code for you to run yourself. Due to the magic
-of adapters, our architecture is mostly independent to your choice of framework.
+Bot frameworks (such as Hubot, Botkit, Rasa or Botpress) provide unique
+approaches to create conversational interfaces and allow developers
+to focus on the design and management of complex bot logic.
+
+Most of framworks can be run on your own bot host. There are also frameworks
```

Comment:
Not true.   Actually they operate as a business on the cloud, charging monthly fees.   This needs to be re-worded.    These frameworks are platforms and may not be "distributed".

Reply:
Fixed. Changed to "operate"